lib/streamlit/DeltaGenerator.py

```diff
@@ -2030,8 +2030,10 @@ def deck_gl_chart(self, element, spec=None, **kwargs):
                   The data for the current layer.
 
                 - "type" : str
-                  A layer type accepted by Deck.GL, such as "HexagonLayer",
-                  "ScatterplotLayer", "TextLayer"
+                  One of the Deck.GL layer types that are currently supported
+                  by Streamlit: ArcLayer, GridLayer, HexagonLayer, LineLayer,
+                  PointCloudLayer, ScatterplotLayer, ScreenGridLayer,
+                  TextLayer.
```

Comment:
I snuck this unrelated docstring improvement into this PR :stuck_out_tongue_winking_eye: